Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender.yml

```diff
@@ -0,0 +1,321 @@
+category: Network Security
+commonfields:
+  id: Microsoft 365 Defender
+  version: -1
+configuration:
+- additionalinfo: The API key to use to connect.
+  defaultvalue: 9093c354-630a-47f1-b087-6768eb9427e6
+  display: Application ID
+  name: app_id
+  required: true
+  type: 4
+- additionalinfo: |-
+    The United States: api-us.security.microsoft.com
+    Europe: api-eu.security.microsoft.com
+    The United Kingdom: api-uk.security.microsoft.co
```

Comment:
if these are all the options we might want to make this a single select field - what do you think?

Reply:
spoke with Arseny about it, he said that we need to allow other options.